lib/svtplay_dl/utils/http.py

```diff
@@ -60,6 +61,28 @@ def download_thumbnail(options, url):
     fd.close()
 
 
+def download_thumbnails(output, config, urls):
```

Comment:
maybe we should replace `download_thumbnail` with this function instead?

Reply:
Yes we could, just wanted to wait until all users of download_thumbnail was converted at the same time. Basically OpenGraphThumbMixIn needs to be changed to use it which all services inherits from, and then I guess all services needs to be tested. For svtplay and barnkanalen I don't use the  OpenGraphThumbMixIn anymore since picks the right thumbnail images directly from the metadata.